### PR TITLE
Polling queue message handler

### PR DESCRIPTION
**Pull request by somesylvie**
# Polling queue message handler
- Added a message handler for polling queue messages
- Trigger queue reader for polling from main
- Move call to sftp handler into polling message handler
- Let Azure Function run from docker-compose
- Added some import message handler tests
- Updated queue message time-to-live settings (on message creation) and visibility timeout settings (on message read)

## Issue
https://github.com/CDCgov/trusted-intermediary/issues/1080

## Checklist

- [x] I have added tests to cover my changes
- [x] I have added logging where useful (with appropriate log level)
- [x] I have added GoDocs where required
- [x] I have updated the documentation accordingly

**Note**: You may remove items that are not applicable

### CODE REVIEW

File: src/sftp/sftp.go [hunk 1]
```diff
@@ -140,12 +142,19 @@ func (receiver *SftpHandler) CopyFiles() {
 		return
 	}
 
+	var wg sync.WaitGroup
```
Comment (pluckyswan) [inline src/sftp/sftp.go:145]: Should we add a comment explaining why we need a WaitGroup?

File: src/orchestration/polling_message_handler.go [hunk 2]
```diff
@@ -2,19 +2,30 @@ package orchestration
...
 }
 
 func (receiver PollingMessageHandler) HandleMessageContents(message azqueue.DequeuedMessage) error {
-	// TODO - use SFTP handler
+	slog.Info("Handling polling message", slog.String("message text", *message.MessageText))
+	// TODO - use the message contents to figure out stuff about config and files
+	// SFTP handler has hard-coded details about where to retrieve files from
+	sftpHandler, err := sftp.NewSftpHandler()
+	if err != nil {
+		slog.Error("ope, failed to create sftp handler", slog.Any(utils.ErrorKey, err))
```
Comment (pluckyswan) [inline src/orchestration/polling_message_handler.go:19]: Did we want to change "ope"?

File: src/sftp/sftp.go (hunk 1 above)
Comment (somesylvie) [reply src/sftp/sftp.go:145]: good call, added a comment

File: src/orchestration/polling_message_handler.go (hunk 2 above)
Comment (somesylvie) [reply src/orchestration/polling_message_handler.go:19]: ope, I guess I should clean up my midwesternisms

**Review by saquino0827 — APPROVED** [review]

**Review by pluckyswan — APPROVED** [review]

**Pull request merged**